catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/Context.java

```diff
@@ -30,6 +31,8 @@
 
     private Optional<Metacard> parentMetacard = Optional.empty();
 
+    private final AtomicBoolean isParentDirty = new AtomicBoolean(false);
```

Comment:
that a child chunk has been uploaded but the parent has not been updated with frame center and location. I found that the shutdown hooks in the monitor can get more than once by osgi, and I needed a way to know if it was just an extra call that could be ignored or if I needed to update the parent because new data came in